opm/parser/eclipse/Deck/UDAValue.hpp

```diff
@@ -23,6 +23,7 @@
 #include <stdexcept>
 #include <vector>
 #include <string>
+#include <iostream>
```

Comment:
```C++
#include <iosfwd>
```
You don't need `<iostream>` here.

Reply:
Fixed - never heard of `isofwd` before.

Reply:
> Fixed

Thanks!

> never heard of `isofwd` before.

The header, sadly, is unknown to many (myself included until fairly recently).  It was created for this exact purpose&mdash;to forward declare IO-stream types used in API declarations so client code wouldn't have to parse the big `<iostream>` header each time they included the header declaring the APIs.  Of course, **using** the APIs in an implementation file will then often require also including `<iostream>` (or the more focused headers `<istream>`, `<ostream>` or `<fstream>`), but it's less of a concern in `.cpp` files.